modules/express/src/expressApp.ts

```diff
@@ -168,7 +169,7 @@ function createHttpServer(app) {
  * @param baseUri
  * @return {Function}
  */
-module.exports.startup = function(config: Config, baseUri: string) {
+export function startup(config: Config, baseUri: string): any {
```

Comment:
return type here is `() => void` (startup returns a function which returns void)